src/main/java/org/sdo/rendezvous/development/DatabaseFiller.java

```diff
@@ -21,39 +21,39 @@
 @Profile("development")
 public class DatabaseFiller {
 
-  private static final String WHITELIST_HASHSET_NAME = "OP_KEYS_WHITELIST";
-  private static final String BLACKLIST_HASHSET_NAME = "OP_KEYS_BLACKLIST";
-  private static final String GUID_BLACKLIST_NAME = "GUIDS_BLACKLIST";
+  private static final String ALLOWLIST_HASHSET_NAME = "OP_KEYS_ALLOWLIST";
+  private static final String DENYLIST_HASHSET_NAME = "OP_KEYS_DENYLIST";
+  private static final String GUID_DENYLIST_NAME = "GUIDS_DENYLIST";
 
   // this key hash has been calculated from public key of
   // /Validation/TestData/OwnershipVoucher/p1.cert.pem
   // instruction how it was calculated can be found
   // /Validation/TestData/OwnershipVoucher/openssl_instructions.md
   // raw data for ecdsa keys are available in file TO0TestData.scala
-  private static final String[] WHITELISTED_KEYS_HASHES_ARRAY = {
+  private static final String[] ALLOWLISTED_KEYS_HASHES_ARRAY = {
     "0734FAC43DBE455D531930B6A8E024043356541BFFCC7A250E417EC38E217725", // rsa pubkey 1
     "3055924C4AF1A77FD365C380F9B3CFC40C5F8C79B1EC6492F0D15648E9792CA2", // rsa pubkey 7
     "8EA241130B68234932411A2B9615B939C70619E4169AFB02D913729456A15C94", // ecdsa384
     "0F600E1E444664FFDEA57B3A83D0CFEB29B406B898142404725874A0CB419AE1" // ecdsa256
   };
 
   // this key hash has been calculated from public key of
-  // /Validation/TestData/OwnershipVoucher/p6_blacklisted.cert.pem
+  // /Validation/TestData/OwnershipVoucher/p6_DENYlisted.cert.pem
   // instruction how it was calculated can be found
   // /Validation/TestData/OwnershipVoucher/openssl_instructions.md
-  private static final String[] BLACKLISTED_KEYS_HASHES_ARRAY = {
+  private static final String[] DENYLISTED_KEYS_HASHES_ARRAY = {
```

Comment:
DENYLIST_KEYS_HASHES_ARRAY

In all cases, use of allowlist or denylist as a verb should be avoided. For variable names, it can be ALLOWLIST_XXX or DENYLIST_XXX.

Reply:
Done